Provide instructions for a Mac client =)

```
$ go get -u github.com/google/key-transparency/cmd/...
# github.com/google/key-transparency/impl/proto/keytransparency_v1_service
.go/src/github.com/google/key-transparency/impl/proto/keytransparency_v1_service/keytransparency_v1_service.pb.go:49: undefined: grpc.SupportPackageIsVersion3
# github.com/google/certificate-transparency/go/merkletree
In file included from .go/src/github.com/google/certificate-transparency/go/merkletree/merkle_tree_go.cc:1:
In file included from ../../cpp/merkletree/merkle_tree.h:10:
In file included from ../../cpp/merkletree/tree_hasher.h:10:
../../cpp/merkletree/serial_hasher.h:4:10: fatal error: 'openssl/sha.h' file not found
```

Homebrew does not let you link the openssl headers anymore:

```
$ brew link openssl --force
Warning: Refusing to link: openssl
Linking keg-only openssl means you may end up linking against the insecure,
deprecated system OpenSSL while using the headers from Homebrew's openssl.
Instead, pass the full include/library paths to your compiler e.g.:
  -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib
```

Additionally, it seems installing https://github.com/grpc/homebrew-grpc is insufficient to solve the grpc undefined error.

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,10 +4,7 @@ go:
   - 1.7.3
 
 dist: trusty
-
-before_install:
-  - sudo apt-get update
-  - sudo apt-get install build-essential libssl-dev
+sudo: false
 
 install:
   - go get -u github.com/kardianos/govendor

